Character.h

```diff
@@ -30,6 +30,7 @@ struct Character
     int getHP() const { return hitPoints; }
     int getArmorLevel() const { return armor; }
     int getAttackDamage() const { return attackDamage; }
+    int getInitialAttackDamage() const { return *initialAttackDamage; }
```

Comment:
- [ ] remove this.  `initialAttackDamage` is protected.  derived classes don't need a member function to access it.

Reply:
`protected:
    std::vector<std::unique_ptr<Item>> defensiveItems;
    std::vector<std::unique_ptr<Item>> helpfulItems;
    int hitPoints, armor;
    int attackDamage;
    bool isDefending = false;
private:
    std::unique_ptr<int> initialHitPoints, initialArmorLevel, initialAttackDamage;
    
    void attackInternal(Character& other);
};`  It's private, not protected, what I find via google is private members of base class are inaccessible to derived class, is that not true?